packages/admin/src/pages/build-application/[applicationId]/dashboard.page.tsx

```diff
@@ -2,23 +2,45 @@ import { GetServerSidePropsContext } from 'next';
 import getConfig from 'next/config';
 import CustomLink from '../../../components/custom-link/CustomLink';
 import Meta from '../../../components/layout/Meta';
-import { getApplicationFormSummary } from '../../../services/ApplicationService';
+import {
+  getApplicationFormSummary,
+  handleSectionOrdering,
+} from '../../../services/ApplicationService';
 import { getGrantScheme } from '../../../services/SchemeService';
 import ServiceError from '../../../types/ServiceError';
 import { getSessionIdFromCookies } from '../../../utils/session';
 import PublishButton from './components/PublishButton';
 import Sections from './components/Sections';
 import UnpublishSummary from './components/UnpublishSummary';
 import InferProps from '../../../types/InferProps';
+import callServiceMethod from '../../../utils/callServiceMethod';
+import { generateErrorPageParams } from '../../../utils/serviceErrorHelpers';
 
 export const getServerSideProps = async ({
   params,
   query,
+  res,
   req,
+  resolvedUrl,
 }: GetServerSidePropsContext) => {
   const { applicationId } = params as Record<string, string>;
   const { recentlyUnpublished } = query;
 
+  const result = await callServiceMethod(
+    req,
+    res,
+    async (body) => {
+      const sectionId = Object.keys(body)[0].split('/')[1];
+      const increment = Object.keys(body)[0].split('/')[0] === 'Up' ? -1 : 1;
```

Comment:
could destructure these from the result of `split` - might make this more readable

Reply:
Fixed in latest commit